packages/rxfire/database/utils.ts

```diff
@@ -26,14 +26,15 @@ export function isNil(obj: any): boolean {
  * that is populated with all the Realtime Database child events.
  * @param events
  */
-export function validateEventsArray(events?: ListenEvent[]) {
+export function validateEventsArray(events?: ListenEvent[]): ListenEvent[] {
+  let validatedEvents: ListenEvent[] = [];
   if (isNil(events) || events!.length === 0) {
```

Comment:
Changed the return type of isNil since it's used elsewhere and this method doesn't touch implementation (playing it safe).  Will use `== null` for future code.

Reply:
You can use generics instead of `any`.

```ts
function isNil<T>(obj: T | null | undefined): obj is null | undefined {
```

I'd still just change it to `== null` though. It's exactly the same as this function. :man_shrugging:

Reply:
Removed isNil and also changed the one other usage in packages/rxfire/database/list/index.ts